### DISCUSSION TITLE
Welcome to opengrok Discussions!

### DISCUSSION THREAD

**Discussion by vladak** [oracle/opengrok#3754] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by d4597** [comment]
Hi , i need to setup opengrok in a server and host source code to it .Can you please guide how to proceed on this clearly

**Reply by vladak** [reply]
The easiest way is to use the [Docker image](https://github.com/oracle/opengrok/blob/master/docker/README.md).
